include/common.h

```diff
@@ -417,6 +417,12 @@ struct GameInfo {
     u8 unkD;
 };
 
+typedef struct {
+    s8 first_byte;
+    u8 padding[95];
+} QuadObj;
```

Comment:
I think this overlaps with an existing type defined here:

https://github.com/sozud/mmx4/blob/766bd9ca0d64f1c289b3419639d25ce1b0b2830f/include/common.h#L296-L299

Reply:
You're right, I'll do a followup